test/cmd/images.sh

```diff
@@ -64,7 +64,7 @@ oc describe istag/mysql:latest
 [ "$(oc describe istag/mysql:latest | grep "Image Created:")" ]
 [ "$(oc describe istag/mysql:latest | grep "Image Name:")" ]
 name=$(oc get istag/mysql:latest --template='{{ .image.metadata.name }}')
-imagename="isimage/mysql@${name:0:7}"
+imagename="isimage/mysql@${name}"
```

Comment:
Yes, the REST API uses the provided image id for image stream images as a
prefix and will find all matches - if there is just one, it is accepted, if
there are multiple, it gives an error.  Basically like Git.

On Wed, Oct 28, 2015 at 3:06 PM, Ben Parees <notifications@github.com>
wrote:

> In test/cmd/images.sh
> <https://github.com/openshift/origin/pull/5409#discussion_r43302985>:
>
> > @@ -64,7 +64,7 @@ oc describe istag/mysql:latest
> >  [ "$(oc describe istag/mysql:latest | grep "Image Created:")" ]
> >  [ "$(oc describe istag/mysql:latest | grep "Image Name:")" ]
> >  name=$(oc get istag/mysql:latest --template='{{ .image.metadata.name }}')
> > -imagename="isimage/mysql@${name:0:7}"
> > +imagename="isimage/mysql@${name}"
>
> Sure, i'm just trying to understand what "retrieve images by their short
> ID" means... is short ID specifically defined, or are you just saying
> "something shorter than the full id"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5409/files#r43302985>.
>